test/blackbox-tests/test-cases/ignore-promoted-rules-internal-rules.t

```diff
@@ -11,7 +11,7 @@ Reported in #8417
   > EOF
 
   $ dune build foo.opam
-  $ echo foobar_extra >> foo.opam
+  $ chmod +w foo.opam && echo foobar_extra >> foo.opam
```

Comment:
I think that can just be added as a separate `$` step as there is no advantage of using `&&` here.